Sources/Datasets/datasets/image/MNIST.swift

```diff
@@ -46,6 +49,13 @@ public struct MNISTDataLoader: S5TFDataLoader {
 
         print("Fetching files... Waiting for the download to finish before continuing...")
         let downloader = Downloader()
+
+        #if os(macOS)
+            let binaryLocation = "/usr/bin/"
+        #else
+            let binaryLocation = "/bin/"
+        #endif
+
         switch split {
             case .train:
                 downloader.download(fileAt: URL(string: baseURL + "/train-images-idx3-ubyte.gz")!,
```

Comment:
I think this needs to be updated.

Reply:
I think that might be an outdated commit, but I'll check.